packaging/rpm/ansible-tower-cli.spec

```diff
@@ -0,0 +1,53 @@
+%{!?python_sitelib: %define python_sitelib %(%{__python} -c "from distutils.sysconfig import get_python_lib; print get_python_lib()")}
+
+Name:           ansible-tower-cli
+Version:        2.3.1
```

Comment:
I'll take a look at parsing the version number out from existing sources (like the VERSION file) to save you the trouble.